doc/man1/dgst.pod

```diff
@@ -8,7 +8,7 @@ dgst, sha, sha1, mdc2, ripemd160, sha224, sha256, sha384, sha512, md4, md5, blak
 
 B<openssl> B<dgst>
 [B<-help>]
-[B<-I<digest>>]
+[B<-[digest]>]
```

Comment:
As for the side note. One can wonder if it's sustainable approach to "synthesize" commands. Maybe it would be more appropriate to "freeze" to options that are documented and are working, and force users to adhere to dgst -algorithm for all other cases...